docs/tutorials/upgrade.md

```diff
@@ -96,8 +96,16 @@ application:
   - Configuration changes:
     - `streaming.forceTransmuxTS` has been renamed to `streaming.forceTransmux`
       (deprecated in v4.3.0)
-    - `manifest.dash.manifestPreprocessor` callback now receives a type of `shaka.externs.xml.Node` instead of `Element`.
-    - `manifest.mss.manifestPreprocessor` callback now receives a type of `shaka.externs.xml.Node` instead of `Element`.
+    - `manifest.dash.manifestPreprocessor` callback now receives a type of
+      `shaka.externs.xml.Node` which is a regular object instead of an `Element`.
+      Attributes and child strings are parsed as is without sanitisation.
+      It is worth noting that use of previous element APIs like `getAttribute`,
+      `textContent`, etc. would have returned html decoded strings.
+    - `manifest.mss.manifestPreprocessor` callback now receives a type of
+      `shaka.externs.xml.Node` which is a regular object instead of an `Element`.
+      Attributes and child strings are parsed as is without sanitisation.
```

Comment:
I'd rather not repeat the comments about `shake.externs.xml.Node`.  Could we factor that out?

Maybe something like:

> - Callbacks that used to accept `Element` now accept `shaka.externs.xml.Node`: `manifest.dash.manifestPreprocessor`, `manifest.mss.manifestPreprocessor`.  `getAttribute()` results ...

Reply:
Good Idea!